_layouts/page.html

```diff
@@ -6,6 +6,16 @@
    <h1 class="post-title-main">{% if page.homepage != true %} {{ page.title }} {% endif %}</h1>
 </div>
 
+<div id="wrap">
+	<div class="container">
+		<a href="https://github.com/pulp" target="_blank" id="fork-me" class='hidden-tablet hidden-phone'><img style="position: fixed;top: 0; right: 0; border: 0;z-index:999999;" src="https://s3.amazonaws.com/github/ribbons/forkme_right_green_007200.png" alt="Fork me on GitHub"></a>
+		<div class="row">
+      <div class="col-md-3 hidden-xs hidden-sm">
+				<div id="toc" class="sidebar-nav-fixed pull-right" data-spy="affix" data-offset-top="60" data-offset-bottom="300">
+				</div>
+			</div>
+
```

Comment:
```suggestion
<div id="wrap">
  <div class="container">
    <a href="https://github.com/pulp" target="_blank" id="fork-me" class='hidden-tablet hidden-phone'><img style="position: fixed;top: 0; right: 0; border: 0;z-index:999999;" src="https://s3.amazonaws.com/github/ribbons/forkme_right_green_007200.png" alt="Fork me on GitHub"></a>
    <div class="row">
      <div class="col-md-3 hidden-xs hidden-sm">
        <div id="toc" class="sidebar-nav-fixed pull-right" data-spy="affix" data-offset-top="60" data-offset-bottom="300">
        </div>
      </div>
```

Hmm, you seem to be opening a lot of div tags, but not closing them.

Reply:
I tried several internet examples, then on preview they all looked like rubbish, so then I robbed this from foreman, and it actually worked. 
I am claiming no expertise here, @mdellweg - I simply want to get the job done. Pragmatism.